### PR TITLE
Fix dataset page & favicon modifications

### PR DESCRIPTION
**Pull request by panveel**
# Description

Fix dataset description & modify the page titles

### CODE REVIEW

File: ckanext/biskit/templates/package/base_form_page.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        {% block form %}
+          {#- passing c to a snippet is bad but is required here
+              for backwards compatibility with old templates and
+              plugins using setup_template_variables() -#}
+          {{- h.snippet(form_snippet, c=c, **form_vars) -}}
+        {% endblock %}
+      {% endblock %}
+    </div>
+  </section>
+{% endblock %}
```
Comment (hkmshb) [inline ckanext/biskit/templates/package/base_form_page.html]:
```suggestion
```
`primary_content` block can be removed as it is an exact replica from the overridden page

File: ckanext/biskit/templates/base.html [hunk 2]
```diff
@@ -1,2 +1,6 @@
 {% resource 'biskit/styles/biskit.css' %}
 {% ckan_extends %}
+
+{% block links -%}
+<link rel="shortcut icon" href="/img/ckan.ico" />
```
Comment (hkmshb) [inline ckanext/biskit/templates/base.html:5]: @panveel can you confirm that this works as I don't see any `ckan.ico` file within `public/img` folder or `fanstatic` folder?

Comment (panveel) [reply ckanext/biskit/templates/base.html:5]: Yes, it's available.

**Review by hkmshb — APPROVED** [review]
LGTM

**Pull request merged**